content/events/skywalkingday-2021/index.md

```diff
@@ -59,6 +59,16 @@ Apache SkyWalking对前端进行了监控与跟踪，分别有Metric, Log, Trace
 2. 了解 “开源≠自由≠非商业” 的来龙去脉。
 3. 一名普通工程师，如何高效地向开源社区做贡献？
 
+#### 如何利用 Apache APISIX 提升 Nginx 的可观测性
+- 金卫 Wei Jin, API7 Engineer Apache SkyWalking committer. Apache apisix-ingress-controller Founder. Apache APISIX PMC.
```

Comment:
Are you replacing Wen Ming's presentation? If so, please remove him from the speaker. Unless @moonming are you going to be a part of some panel?

Reply:
please remove me

Reply:
ok